virttest/qemu_monitor.py

```diff
@@ -22,13 +23,17 @@
     logging.warning("Could not import json module. "
                     "QMP monitor functionality disabled.")
 
-from . import passfd_setup
 from . import utils_misc
 from . import cartesian_config
 from . import data_dir
 
 from virttest.qemu_capabilities import Flags
 
+if sys.version_info[:2] >= (3, 3):
```

Comment:
Well this way the 3.0 - 3.2 would not be covered as the requirements condition is `<3.0` but the `array` is only available since `3.3`. Anyway Avocado itself requires `>3.4` so I'd suggest to just use `sys.version[0] >= 3` and not to trouble about the 3.0 - 3.3. What do you think? (maybe to be super cautious, let's add the `>=3.3` to requirements, but it shouldn't be needed as Avocado already limits this)

Reply:
Make sense to me.